spec/models/user_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+describe User do
+  it { is_expected.to validate_presence_of(:first_name) }
+  it { is_expected.to validate_presence_of(:last_name) }
+
+  subject(:user) do
+    build(:user)
+  end
+
+  it do
```

Comment:
This example is unnecessary because a factory always must return a valid object!